sklearn/metrics/cluster/adjacency_matrix.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+
+def adjacency_matrix(cluster_assignement):
```

Comment:
I did not find the adjacency matrix in metrics.cluster.supervised, in contingency matrix or other functions.

The adjusted_rand_score makes some calculations that are related to the fowlkes mallows index, for which we use adjacency matrix, but I did not find the way to remove the "adjacency matrix" code